### PR TITLE
add vendor - google uuid

### PR DESCRIPTION
**Pull request by boqiu**

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -39,6 +39,12 @@
 			"revision": "c2828203cd70a50dcccfb2761f8b1f8ceef9a8e9",
 			"revisionTime": "2018-01-10T05:33:47Z"
 		},
+		{
```
Comment (WTer) [inline vendor/vendor.json]: Why did you add this section?

Comment (boqiu) [reply vendor/vendor.json]: This is auto-generated by govendor which a standard tool to import packages into project.

Comment (WTer) [reply vendor/vendor.json]: got it, but its revisionTime is a little strange.
